tests/aws/services/iam/test_iam.py

```diff
@@ -34,8 +35,10 @@ def test_get_user_without_username_as_user(self, create_user, aws_client):
         )
         account_id = aws_client.sts.get_caller_identity()["Account"]
         keys = aws_client.iam.create_access_key(UserName=user_name)["AccessKey"]
-        wait_for_user(keys)
-        iam_client_as_user = create_client_with_keys("iam", keys=keys)
+        wait_for_user(keys, TEST_AWS_REGION_NAME)
```

Comment:
Not a fan of using the constants here. We could just use `aws_client.iam.meta.region_name` or something, so it always matches the region of the client.

Reply:
Replaced with the `region` fixture